webpack/scenes/ContentViews/Table/ContentViewsTable.js

```diff
@@ -26,15 +26,12 @@ const ContentViewTable = ({ response, status, error }) => {
   useEffect(
     () => {
       if (!loadingResponse && results) {
-        const { updatedRowMap, ...tableData } = tableDataGenerator(
-          results,
-          rowMapping,
-        );
+        const { newRowMappingIds, ...tableData } = tableDataGenerator(results);
         setTable(tableData);
-        setRowMapping(updatedRowMap);
+        setRowMappingIds(newRowMappingIds);
       }
     },
-    [results, JSON.stringify(rowMapping)], // use JSON to check obj values eq not reference eq
+    [results, JSON.stringify(rowMappingIds)], // use JSON to check obj values eq not reference eq
```

Comment:
Same comment from before about the `JSON.stringify`

Reply:
We don't need to check rowMappingIds in the dependency array since the value is generated from results in the first place. I removed it.